content/release-10-15-0/whatsnew-10-15-0.md

```diff
@@ -11,3 +11,40 @@ Release 10.15.0 includes the following new features or major feature enhancement
 In an effort to provide you a more consistent experience across the platform the **All devices** page has been re-shaped. It is now implemented in the data grid fashion you already are familiar with from **Subassets** and **Search** pages. This offers improved appearence in various viewport sizes, easier filtering and sorting, pagination option (only for users with global inventory access) and sets solid ground for further improvements.
 
 ![All devices](/images/release-notes/devmgmt-devices-alldevices.png)
+
+### Digital Twin Manager
+
+Cumulocity IoT Digital Twin Manager is a layered product hosted on Cumulocity platform. The Digital Twin Manager enables you to create and manage assets around your physical connected devices in Cumulocity IoT. Assets can be used to structure and describe devices in logical hierarchies as encountered in the real world environment.
+
+The Digital Twin Manager consists of several components, which support you to create your assets and asset hierarchies:
+
+    •	Property library - Create your own custom properties (simple or complex) to define how data should be stored on an asset
+    •	Asset types - Define asset types to describe the blueprint of your assets
+    •	Localization - Dynamically translate asset type and custom property descriptions to multiple languages
+    •	Assets navigator - Manage your asset hierarchies
+
+You can access Digital Twin Manager from the Cumulocity IoT app switcher:
+
+![App switcher](/images/dtm/app-switcher-dtm.png)
```

Comment:
This image is not of much value. I would better show the DTM here.

Reply:
Is it DTM landing screen?

Reply:
Yes, I think that is better. Showing the app switcher does not make much sense.

Reply:
Then I will add the DTM landing page screenshot

Reply:
I have updated the image and committed. Please review and let me know if anything needs to be modified